meshtastic/interfaces/ble/errors.py

```
@@ -104,6 +105,47 @@ def _safe_execute(
                 raise
             return default_return
 
+    # COMPAT_STABLE_SHIM: Public compatibility alias; delegates to _safe_execute.
+    @staticmethod
+    def safe_execute(
+        func: Callable[[], T],
+        default_return: T | None = None,
+        log_error: bool = True,
```

Ruff (0.15.5)

[warning] Boolean-typed positional argument in function definition

(FBT001)